### DISCUSSION TITLE
Broken layout when downloading from virtual machine (linux) into windows

### DISCUSSION THREAD

**Discussion by AlexTeufel** [quarto-dev/quarto-cli#10079] [Q&A] [answered]
### Description

Hello, i've been trying to use Quarto for first time, its running fine on my virtual machine, but when i download it and try to see it on windows firefox the layout seems to be broken. Anyone knows how to fix it?

Im using https://github.com/jjallaire/gapminder-dashboard/blob/main/gapminder.qmd  as an example. (im just copy+pasting it)

On my virtual machine using VS Code:
![image](https://github.com/quarto-dev/quarto-cli/assets/110360151/dab2221c-f65f-48bb-a320-a15f75c4e7b2)

When downloading the html and trying to watch it on my local machine: 
![image](https://github.com/quarto-dev/quarto-cli/assets/110360151/dce6ba2e-3cbf-419c-b1fb-ccb5853ad4b2)

What im doing wrong?

**Comment by mcanouil** [comment, marked as answer]
You need to download the assets associated with the HTML file.
A directory like `*_files`.

Or set `embed-resources: true` in the YAML frontmatter.
